site/src/util/deployOptions.ts

```diff
@@ -0,0 +1,34 @@
+import { DeploymentGroup, DeploymentOption } from "./../api/types"
+
+export const findDeploymentOptions = (
+  options: DeploymentOption[],
+  ...names: string[]
+): DeploymentOption[] => {
+  const found: DeploymentOption[] = []
```

Comment:
Wrap this in a `useMemo` hook to prevent searching from hundreds of options on re-render!